haswell/lib_haswell_memory_bound.cpp

```diff
@@ -102,7 +120,12 @@ int haswell_membound_active_cycles(metric_id_t metric_id,
     // Update the counter values at the current sample period
     update_values(metric_id, current_sample_time);
 
-    *out_value= gEventValues.at(EventInds::CLK_UNHALTED_IND);
+    *out_value= 0;
```

Comment:
If you don't write to *out_value at all then MAP will treat the value as not present rather than zero, which you may prefer.

Reply:
Yes, that is preferred. I will fix it.